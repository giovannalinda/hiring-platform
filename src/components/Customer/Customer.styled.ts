import styled from 'styled-components'
import { theme } from '~/styles'

export const Container = styled.section`
  background: ${theme.colors.green200};
  border-radius: ${theme.borderRadius.lg};
  text-align: center;
  padding: ${theme.space[7]};
  margin-bottom: 12rem;
`

export const TextGroup = styled.div`
  padding-top: 6.2rem;
  padding-bottom: 6.2rem;
  display: flex;
  align-items: center;
  justify-content: center;
  flex-direction: column;

  h1 {
    font-size: ${theme.font.sizes.title};
    color: ${theme.colors.black};
    padding-bottom: ${theme.space[7]};
  }

  p {
    color: ${theme.colors.black};
    font-size: ${theme.font.sizes.paragraph};
    max-width: 46.4rem;
  }
`

export const Content = styled.div`
  display: flex;
  background: ${theme.colors.black};
  border-radius: ${theme.borderRadius.lg};
  margin-bottom: ${theme.space[7]};

  @media screen and (max-width: ${theme.breakpoints.tablet}) {
    flex-direction: column;
  }
`

export const ContentText = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: center;
  text-align: left;
  color: ${theme.colors.white};
  max-width: 70rem;
  padding: ${theme.space[7]};

  h2 {
    font-size: 3rem;
    padding-bottom: ${theme.space[7]};
  }

  h3 {
    font-size: ${theme.font.sizes.paragraph};
    padding-bottom: ${theme.space[1]};
  }

  p {
    font-size: ${theme.font.sizes.paragraph};
    color: ${theme.colors.gray500};
  }
`
