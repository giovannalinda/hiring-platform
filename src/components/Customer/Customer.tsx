import Image from 'next/image'
import { Assets } from '~/assets'
import * as S from './Customer.styled'

export function Customer() {
  return (
    <S.Container>
      <S.TextGroup>
        <h1>What our beloved customers says</h1>
        <p>
          We have over 10.000 companies that use our tools and services, hear
          their story
        </p>
      </S.TextGroup>

      <S.Content>
        <Image src={Assets.BusinessWoman} width={393} height={387} />
        <S.ContentText>
          <h2>
            “Since using Recruiter, our workflow has improved dramatifally, the
            tool makes HR very agile and lightweight”
          </h2>
          <h3>Anna Smith</h3>
          <p>Head of Human Resources Department</p>
        </S.ContentText>
      </S.Content>
    </S.Container>
  )
}
