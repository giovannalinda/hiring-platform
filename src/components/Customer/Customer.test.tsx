import { render } from '~/__helpers__/app-tests'
import { Customer } from './Customer'

describe('Customer', () => {
  it('should match snapshot', () => {
    const { container } = render(<Customer />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
