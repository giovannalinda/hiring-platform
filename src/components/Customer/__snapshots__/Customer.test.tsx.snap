// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Customer should match snapshot 1`] = `
<section
  class="sc-bczRLJ hdfkzd"
>
  <div
    class="sc-gsnTZi kKPEBo"
  >
    <h1>
      What our beloved customers says
    </h1>
    <p>
      We have over 10.000 companies that use our tools and services, hear their story
    </p>
  </div>
  <div
    class="sc-dkzDqf kZQgJE"
  >
    <img
      height="387"
      src="/BusinessWoman.svg"
      width="393"
    />
    <div
      class="sc-hKMtZM kewfce"
    >
      <h2>
        “Since using Recruiter, our workflow has improved dramatifally, the tool makes HR very agile and lightweight”
      </h2>
      <h3>
        Anna Smith
      </h3>
      <p>
        Head of Human Resources Department
      </p>
    </div>
  </div>
</section>
`;
